test.js

```diff
@@ -9,7 +9,15 @@ test('should replace `data-mc-edit` tags with `mc:edit`', function (t) {
   t.end();
 });
 
-test('should return exact string if no `data-mc-edit` is present', function (t) {
+test('should replace `data-mc-repeatable` tags with `mc:repeatable', function (t) {
+  var html = '<div data-mc-repeatable>foobaz</div>';
```

Comment:
What if `data-mc-repeatable="foo"` is passed in? I don't think Mailchimp supports values for `mc:repeatable` and might just toss it out.